hidden/Common/Constant.swift

```diff
@@ -15,6 +15,7 @@ let IS_SHOW_PREFERENCES = "isShowPreferences"
 let IS_KEEP_LAST_STATE = "isKeepLastState"
 let App_Name = "Hidden Bar"
 let IS_COLLAPSE = "isCollapse"
+let IS_PERM_HIDE_ENABLED = "isPermHideEnabled"
```

Comment:
Please move this to extension of Notification.Name, put it at global scope not so good, we can't move the old keys since it related to existed users. I also prefer value format like this:  "NUMBER_OF_SECOND_FOR_AUTO_HIDE"

Reply:
Solved in latest commit